docs/general/faq.md

```diff
@@ -15,9 +11,9 @@ Update to Node.js 14.0.0 or newer.
 ## How do I install FFMPEG?
 
 - **npm:** `npm install ffmpeg-static`
-- **Ubuntu 16.04:** `sudo apt install ffmpeg`
-- **Ubuntu 14.04:** `sudo apt-get install libav-tools`
-- **Windows:** `npm install ffmpeg-static` or see the [FFMPEG section of AoDude's guide](https://github.com/bdistin/OhGodMusicBot/blob/master/README.md#download-ffmpeg).
+- **Ubuntu 20.04:** `sudo apt install ffmpeg`
+- **Ubuntu 18.04:** `sudo apt install ffmpeg`
```

Comment:
```suggestion
- **Ubuntu 18.04+:** `sudo apt install ffmpeg`
```

Reply:
...no. The only reason those files changed is because I rebased master. Nothing to do with this PR.